test/checks/isQuadratic.test.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const assert = require('assert');
+const math = require('mathjs');
+
+const flatten = require('../../lib/util/flattenOperands');
+const checks = require('../../lib/checks');
+
+function testIsQuadratic(exprString, result) {
```

Comment:
I just merged an update to add `testUtil`, so these tests should be of the new format now